src/containers/HomePageSecondPart/HomePageSecondPart.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
why are u even making this component ? u can just render SearchBar component directly inside the HomePage container